packages/components/tests/acceptance/percy-test.js

```diff
@@ -155,6 +155,10 @@ module('Acceptance | Percy test', function (hooks) {
     await visit('/utilities/dismiss-button');
     await percySnapshot('DismissButton');
 
+    // MOVE THIS BLOCK IN THE RIGHT POSITION
```

Comment:
Don't forget to remove this line and group the snapshot with the other form components

Reply:
Yep, just noticed that earlier as I was going through refactoring.